### DISCUSSION TITLE
galaxy brain

### DISCUSSION THREAD

**Discussion by lironmiz** [drknzz/GitHub-Achievements#47] [Q&A]
hii how are you?

**Comment by lironmiz** [comment]
this good

**Comment by drknzz** [comment]
To get the Galaxy Brain achievement, please answer these 2 discussions:
- [Discussion 1](https://github.com/drknzz/GitHub-Achievements/discussions/48)
- [Discussion 2](https://github.com/drknzz/GitHub-Achievements/discussions/49)

**Comment by DanielR-spec** [comment]
Thank u for ur support :100:

**Comment by the1Riddle** [comment]
i am good

**Comment by bettyc0der** [comment]
please give me galaxy brain and star my repo I am student thanks!
https://github.com/salcedocharlotte/CIS261-Course-Project
